config/backend-config/backend-config.go

```diff
@@ -329,13 +330,19 @@ func (bc *backendConfigImpl) StartWithIDs(ctx context.Context, workspaces string
 		u, _ := identifier.BasicAuth()
 		secret := sha256.Sum256([]byte(u))
 		cacheKey := identifier.ID()
-		bc.cache = cache.Start(
+		bc.cache, err = cache.Start(
 			ctx,
 			secret,
 			cacheKey,
 			bc.Subscribe(ctx, TopicBackendConfig),
 		)
+		if err != nil {
```

Comment:
@Sidddddarth is it really a valid scenario to have rudder-server running in an environment without a jobsdb?
From what I can tell, at least in production, all pods, including warehouse, have `JOBS_DB` environment variables set.
So the question is, should we proceed by disabling cache, or panic?

Reply:
I think we can leave the panic to other components.
But I agree, the premise around the entire thing was that without jobsdb the server crashes anyway, so we never really pursued this idea.